packages/typespec-client-generator-core/src/public-utils.ts

```diff
@@ -63,7 +63,7 @@ export function getDefaultApiVersion(
  * @param parameter
  * @returns
  */
-export function isApiVersion(context: TCGCContext, type: { name: string }): boolean {
+export function isApiVersion(type: { name: string }): boolean {
```

Comment:
breaking change: if i can figure out how to better check for core params, then won't need to make this change